client/apps/edit/components/section_list/index.coffee

```diff
@@ -44,6 +44,7 @@ module.exports = React.createClass
               index: i
               editing: @state.editingIndex is i
               ref: 'section' + 1
+              key: section.cid
```

Comment:
Initially I tried using `i` but that didn't help as it wasn't a property that carried with the section as it moved around. So good to know that this key needs to be unique among the array, _as well_ as specific to the data you're passing into the component.